.github/workflows/release_portable_linux_packages.yml

```
@@ -61,6 +61,8 @@ on:
 permissions:
   contents: read
 
+run-name: Release portable Linux packages (${{ inputs.families }}, ${{ inputs.release_type }})
```

Most of the other workflows got run names set as expected. This one and `.github/workflows/release_windows_packages.yml` did not, since they run on `schedule` and compute their inputs further down in the file
https://github.com/ROCm/TheRock/actions/runs/19917417466

```yml
    env:
      release_type: ${{ inputs.release_type || 'nightly' }}
```

```yml
      - name: Generating package target matrix
        id: configure
        env:
          AMDGPU_FAMILIES: ${{ inputs.families }}
          THEROCK_PACKAGE_PLATFORM: "linux"
        run: python ./build_tools/github_actions/fetch_package_targets.py
```

Maybe we can change the text to "(default, nightly)" if the values are unset

This might work?

```suggestion
run-name: Release portable Linux packages (${{ inputs.families || 'default' }}, ${{ inputs.release_type || 'nightly' }})
```

---

Seems to work. Sent https://github.com/ROCm/TheRock/pull/2451 to fix